docs/guides/qiskit-serverless.mdx

```diff
@@ -1,232 +1,32 @@
 ---
-title: Run workloads remotely with Qiskit Serverless
-description: How to run quantum computing workloads remotely using Qiskit Serverless on IBM Quantum.
+title: Qiskit Serverless overview
+description: Use Qiskit Serverless to run quantum-classical workloads across cloud and quantum-centric supercomputing environments.
 ---
 
-# Run workloads remotely with Qiskit Serverless
+# What is Qiskit Serverless?
 
-Premium users can build, deploy, and run their workloads remotely on classical compute made available through the IBM Quantum&trade; Platform.
-
-Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account) and explore more of the features of Qiskit Serverless in the [documentation](https://qiskit.github.io/qiskit-serverless/).
+Qiskit Serverless provides a simple interface to run workloads across quantum-classical resources. This includes deploying programs to the IBM Quantum Platform, running workloads remotely, and easy resource management across cloud and quantum-centric supercomputing environments.
 
+Premium users can build, deploy, and run their workloads remotely on classical compute made available through the IBM Quantum Platform. Try out the tutorials in [IBM Quantum Learning](https://learning.quantum.ibm.com/catalog/tutorials?topics=qiskit-patterns) (note: these are accessible in the Premium Plan once you have logged into your IBM Quantum account).
 
 <Admonition type="note">
-    This is an experimental feature, subject to change.
+    This is a preview feature, subject to change.
```

Comment:
ugh I'm really not a fan of this "preivew" term, it means nothing to users, is there any way we can explain this term a bit more?

Reply:
I think we can change how we phrase it, as long as we stay to the spirit of "preview"

Reply:
How about something like "This feature is under continual development, subject to change" (or "frequent development")